src/components/RadioButtonGroup/readme.md

```diff
@@ -268,3 +268,187 @@ class ErrorRadioButtonGroup extends React.Component {
+        name: 'Tahimi',
+        email: 'tahimi@gmail.com',
+        linkedin: '#',
+        github: '#',
+        isActive: false,
+    }
+];
+
+const Title = styled.h1.attrs(props => props.theme.rainbow)`
+    font-family: 'Lato';
+    font-size: 36px;
+    text-align: center;
+    color: ${props => props.palette.text.main};
+`;
+
+const StyledAvatar = styled(Avatar)`
+    width: 100px;
+    height: 100px;
+    box-shadow: 0 2px 4px 0 rgba(6, 28, 63, 0.3);
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+`;
+
+const NameLabel = styled.label.attrs(props => props.theme.rainbow)`
+    font-family: Lato;
+    font-size: 16px;
+    font-weight: 900;
+    text-align: center;
+    color: ${props => props.palette.text.main};
+    display: block;
+`;
+
+const EmailLabel = styled.span.attrs(props => props.theme.rainbow)`
+    font-family: Lato;
+    font-size: 12px;
+    font-weight: 300;
+    text-align: center;
+    color: ${props => props.palette.text.label};
+    display: block;
+`;
+
+const ConctactContainer = styled.div.attrs(props => props.theme.rainbow)`
+    width: max-content;
+`;
+
+const Contact = (props) => {
+    const {avatar, name, email, linkedin, github} = props;
+    return (
+        <ConctactContainer className="rainbow-m-around_medium">
+            <StyledAvatar
+                src={ avatar }
+                assistiveText={ name }
+                title={ name }
+            />
+            <NameLabel className="rainbow-m-top_small">
+                { name }
+            </NameLabel>
+            <EmailLabel className="rainbow-m-top_x-small">
+                { email }
+            </EmailLabel>
+            <div className="rainbow-flex rainbow-justify_space-around rainbow-m-top_small">
+                <a href={linkedin}>
+                    <GitHubIcon />
+                </a>
+                <a href={github}>
+                    <LinkedinIcon />
+                </a>
+            </div>
+        </ConctactContainer>
+    );
+};
+
+const Contacts = ({data}) => {
+    const [showActive, setShowActive] = useState('active');
+
+    const options = [
+        { value: 'active', label: 'Actives' },
+        { value: 'all', label: 'All' },
+    ];
+
+    const handleOnChange = ( event ) => {
+        setShowActive(event.target.value);
+    }
+    
+    return (
+        <div className="rainbow-flex rainbow-flex_column rainbow-align_center">
+            <RadioButtonGroup
+                className="rainbow-m-top_large rainbow-m-bottom_medium"
+                options={options}
+                value={showActive}
+                variant="brand"
+                onChange={handleOnChange}
+            />
+            <div className="rainbow-flex rainbow-flex_wrap rainbow-justify_space-around">
+                {data.filter(contact => showActive === 'all' || contact.isActive).map((contact, i) => {
+                    return (
+                        <Contact
+                            key={i}
```

Comment:
Use index instead of i, let's try to avoid using variables names with only one letter to improve readability.
Also react normally waring you when using index direct as a list key, instead, construct one:
```
const key = `contact-${index}`;
<Contact
    key={key}
    ...
```